src/hooks/ProcessedRichNote/ProcessedRichNote.ts

```diff
@@ -1,10 +1,17 @@
 import { useEffect, useMemo, useState } from 'react'
+import { formatIpfsLink } from 'utils/ipfs'
 
 import { ProjectPreferences } from 'constants/v2/projectPreferences'
 import { loadAllMediaLinksPerLine } from './loadAllMediaLinksPerLine'
 
+// Gets strings that start with 'https'
 const URLRegex = new RegExp(
-  /((?:https?):\/\/(?:\w+:?\w*)?(?:\S+)(:\d+)?(?:\/|\/([\w#!:.?+=&%!\-/]))?)/gi,
+  /((?:https|ipfs?):\/\/(?:\w+:?\w*)?(?:\S+)(:\d+)?(?:\/|\/([\w#!:.?+=&%!\-/]))?)/gi,
```

Comment:
**question** should this remain unchanged?

Reply:
Ahh yep it should! My bad